What-I've-Learned.md

```diff
@@ -3,3 +3,25 @@ layout: link
 title: What I've Learned
 ---
 ## My time at KUB has been teaching me many things. I've always been intrested in computers but I've actually never coded before. Here's some of the things I've learned. Coding has a lot of commands and things you learn so you can code efficiently. Here's a list:
+cd: This is change directory. I used this to navigate through folders stored in my computer!
+
+git clone: I used this command to clone other repository into my directory.
+
+git log: This commands showed me all of my changes that are called commits!
+
+git checkout: On my computer I have a main branch that holds my current code. You make branches to make changes without affecting main. This commands lets me change which branch I'm coding on!
+
+gnew: Command to make a new branch!
+
+git add ., git commit, git push, and git merge: This may be a little long because these commands work in a process. Git add adds all your changes into staged changes to get them ready to 
+push. Git commit is the command that turns those stage changes into commits! Commits are latest changes that ready to push to the current build or project. Git push is just pushing those changes into your main branch. Lastly, git merge is the command to merge your latest changes into the main branch!
+
+--force: Usually don't use this command but its used to force changes to push if there's a error with a command like git push.
+
+gup: This command shows you the latest verison of the branch you are currently on.
```

Comment:
This command doesn't really _show_ you anything.  When I read this, it gives me the impression that some sort of text is displayed, but gup  brings your code up to date with the code in github. Could you reword this?